carbon-design-system/carbon-addons-iot-react
# StatefulTable find rowAction for nested rows

Closes  #1363

**Summary**

- Added a function to include nested rows whilst traversing if a rowAction is triggered.

**Acceptance Test (how to verify the PR)**

- Go to storybook
- Under `Table` click on `stateful example with row nesting and fixed columns`
- Click on any `rowAction` of a nested row
- Check console - no errors should appear
- Check Action knob - correct action should be displayed
